style.css

```diff
@@ -614,13 +621,14 @@
 
   .work-container {
     width: 100%;
-    margin: 0;
+
+    /* margin: 0; */
```

Comment:
I think it will be better to get rid of unused code instead of commenting them. This increases processing time and also undetermined errors.